src/components/RightMenu.tsx

```diff
@@ -33,17 +35,21 @@ const items: SelectItem<ContractsENV>[] = [
   },
 ]
 
-if (ADD_LOCAL_NETWORK) {
-  items.push({
-    label: 'Local',
-    value: 'local',
-    icon: <TestnetIcon />,
-  })
+const explorerUrls: Record<string, string> = {
+  dar: 'https://dar.explorer.fluence.dev',
+  kras: 'https://explorer.fluence.dev',
 }
 
+// if (ADD_LOCAL_NETWORK) {
```

Comment:
why? is it possible to add somehow custom chain URL and indexer URL for local dev in ur PR?